docs/basics.md

```diff
@@ -254,3 +255,35 @@ In this example, `model` and `something` are directly available on the
 `MyObject` instance, while `another` must be accessed via `getOption`. This is
 handy when you want to add extra keys that will be used heavily throughout the
 defined class.
+
+### The `options` Property
+
+The Marionette classes accept an `options` property in the class definition
+which is merged with the `options` argument passed at instantiation. The 
+values from the passed in `options` overrides the property values.
+
+> The `options` argument passed in `initialize` method is equal to the passed at 
+> class instantiation. To get the option inside initialize considering the 
+> `options` property is necessary to use `getOption`  
```

Comment:
Do you need the `>` here?

Reply:
The idea is to highlight the behavior which is being changed by this PR

Reply:
I think the general docs should really be present timelessly and not indicate what is new

Reply:
This excerpt does not have a reference of the change between versions or even that had a different behavior so is time lees. The objective is to highlight an aspect that could be wrongly interpreted by reading the text above where it says that the options are merged, so the user could think that this is the case for the options passed in initialize.

Anyway if there's objection i can remove the remark

Reply:
works for me